civihr_employee_portal/js/reports.js

```diff
@@ -851,7 +882,9 @@
               }
 
               if (REPORT_NAME === 'leave_and_absence') {
-                initCurrentAbsencePeriodFilterDates().then(resolve, reject);
+                formFiltersStore.values.fromDate = moment().subtract(1, 'months').toDate();
```

Comment:
is `initCurrentAbsencePeriodFilterDates` still used anywhere?

Reply:
Nope, I guess I'll remove it.